tests/test_embed/test_interpreter.cpp

```diff
@@ -197,6 +197,22 @@ TEST_CASE("Custom PyConfig with argv") {
         const auto &cpp_widget = py_widget.cast<const Widget &>();
         REQUIRE(cpp_widget.argv0() == "a.out");
     }
+
+    {
+        PyConfig config;
+        PyConfig_InitPythonConfig(&config);
+
+        // Parse arguments the way python normally does, and strip the first argument (which is
```

Comment:
Suggested wording to clearly bring out the essence:

```
// `initialize_interpreter() overrides the default value for config.parse_argv (`1`) by changing it to `0`.
// This test exercises `scoped_interpreter` with the default.
```

Reply:
Amended in 20b138d77c002973ad369c30c5bd56fde24f2232